### PR TITLE
Add `vscode:` URI prefix to knownSchemes in links.ts

### LINKED ISSUE #71414: Markdown knownSchemes lacks vscode's own URI scheme
Issue Type: <b>Bug</b>

## Issue
Create a Markdown file that contains a link with the `vscode:` URI scheme:
```
[Link to Python extension for vscode](vscode:extension/ms-python.python)
```

Expected link location to be `vscode:extension/ms-python.python`, and displayed as [Link to Python extension for vscode](vscode:extension/ms-python.python).

Actual link location `vscode-resource:/vscode:extension/ms-python.python`, and displayed as [Link to Python extension for vscode](vscode-resource:/vscode:extension/ms-python.python).

## Proposed Solution
Add `'vscode:'` to `knownSchemes` array in [extensions/markdown-language-features/src/util/links.ts](https://github.com/Microsoft/vscode/blob/f5044f0910e4aa7e7e06cb509781f3d56e729959/extensions/markdown-language-features/src/util/links.ts#L8).

VS Code version: Code 1.32.3 (a3db5be9b5c6ba46bb7555ec5d60178ecc2eaae4, 2019-03-14T22:46:48.547Z)
OS version: Darwin x64 18.0.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz (4 x 2700)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 4|
|Memory (System)|8.00GB (0.02GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (34)</summary>

Extension|Author (truncated)|Version
---|---|---
rtf|ale|2.0.1
solargraph|cas|0.19.3
vscode-modelines|chr|2.0.2
npm-intellisense|chr|1.3.0
vscode-npm-script|eg2|0.3.5
vscode-diff|fab|1.4.0
vscode-git-history|fab|2.0.5
beautify|Hoo|1.4.11
vscode-duplicate|mrm|1.2.1
python|ms-|2019.3.6139
cpptools|ms-|0.22.1
csharp|ms-|1.18.0
mono-debug|ms-|0.15.8
incrementor|nms|0.1.0
printcode|nob|3.0.0
openhab|ope|0.4.1
vscode-docker|Pet|0.6.1
platformio-ide|pla|1.6.0
diff|raf|0.0.1
ruby|reb|0.22.3
java|red|0.41.0
LiveServer|rit|5.5.1
partial-diff|ryu|1.4.0
code-settings-sync|Sha|3.2.7
code-spell-checker|str|1.6.10
unity-debug|Uni|2.7.2
vscodeintellicode|Vis|1.1.5
vscode-arduino|vsc|0.2.25
vscode-java-debug|vsc|0.17.0
vscode-java-dependency|vsc|0.4.0
vscode-java-pack|vsc|0.6.0
vscode-java-test|vsc|0.15.1
vscode-maven|vsc|0.15.2
markdown-all-in-one|yzh|2.2.0

(1 theme extensions excluded)

</details>

### PR DESCRIPTION
**Pull request by phaseOne**
I have not yet built and tested this branch. Should fix #71414.

### CODE REVIEW

**Comment by phaseOne** [comment]
Well, this fixes the link output issue, but clicking it does not launch the vscode page.

File: extensions/markdown-language-features/src/util/links.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 import * as vscode from 'vscode';
 
-const knownSchemes = ['http:', 'https:', 'file:', 'mailto:', 'data:', 'vscode-resource:'];
+const knownSchemes = ['http:', 'https:', 'file:', 'mailto:', 'data:', 'vscode:', 'vscode-resource:'];
```
Comment (mjbvz) [inline extensions/markdown-language-features/src/util/links.ts]: Please also add `vscode-insiders` so insiders links also work

Comment (phaseOne) [reply extensions/markdown-language-features/src/util/links.ts]: Makes sense

**Comment by mjbvz** [comment]
Thanks!

**Pull request merged**